src/library/autodj/dlgautodj.ui

```diff
@@ -219,6 +219,13 @@
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="QLabel" name="labelTotalInfo">
```

Comment:
All other features have the text left aligned. I think we should do it here as well.

Reply:
Did you consider this comment?

Reply:
Well, `labelSelectionInfo` does not have text alignment set, so I'm slightly confused why `labelTotalInfo` would be different.

Reply:
This is because in the analysis view the spacer is on the very right. 
![grafik](https://user-images.githubusercontent.com/1777442/235293351-49e64d68-20b6-4458-a645-c9bc117217fd.png)
I have no strong preferences for one of the options but they should be the same.